aait/backend/group-2A/WebApi/Controllers/LikeController.cs

```diff
@@ -0,0 +1,49 @@
+using Application.Features.Like.Request.Commands;
+using Application.Features.Like.Request.Queries;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Application.DTO.UserDTO;
+
+namespace WebApi.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class LikeController : ControllerBase
+    {
+        private readonly IMediator _mediator;
+
+        public LikeController(IMediator mediator)
+        {
+            _mediator = mediator;
+        }
+
+        [HttpPost("LikePost")]
+        public async Task<IActionResult> LikePost(CreateLikeCommand command)
+        {
+            await _mediator.Send(command);
+            return NoContent();
+        }
+
+        [HttpPost("UnlikePost")]
+        public async Task<IActionResult> UnlikePost(DeleteLikeCommand command)
```

Comment:
pass an Id to unlikePost function or deleteLikeDto instead of DeleteLikeCommand and also for delete use HttpDelete